api/src/routers/meals.js

```diff
@@ -1,8 +1,31 @@
 import express from "express";
 import knex from "../database_client.js";
+import dayjs from "dayjs";
 
 const meals = express.Router();
 
+meals.post("/add", async (req, res) => {
+  try {
+    const { title, description, location, when, max_reservations, price, img_url } = req.body;
+    const formattedDate = dayjs(when).format("YYYY-MM-DD HH:mm:ss");
```

Comment:
Can this conversion fail? Then we probably want to send a Bad Request response back :) In general you would like to handle the error cases yourself so you help out the client side.

Reply:
Yes it can fail for sure , I have added validation. Thank you for the comment  :)